scripts/devportal/update-themes-and-modules.sh

```diff
@@ -25,33 +30,53 @@ function add_all_repositories {
 function add_all_modules_repositories {
     for repo_name in ${modules[@]}
     do
-        git remote add -f ${repo_name} git@bitbucket.org:pronovix/${repo_name}.git
+      register_repository ${repo_name}
     done
 }
 
 function add_all_themes_repositories {
     for repo_name in ${themes[@]}
     do
-        git remote add -f ${repo_name} git@bitbucket.org:pronovix/${repo_name}.git
+      register_repository ${repo_name}
     done
 }
 
+function register_repository {
+  local repo_name=$1
+  git remote get-url ${repo_name} &> /dev/null
+  if [[ $? -gt 0 ]]; then
+    git remote add -f ${repo_name} ${DEVPORTAL_REMOTE_REPOSITORY_URL}/${repo_name}.git
+  fi
```

Comment:
So what's gonna happen if the directory name is NOT a repo name? Will this directory be skipped silently?

Reply:
You get an error from GIT that the repository does not exist but the script continues to run.